conductor-client/src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -704,7 +704,7 @@ class PostgresEntityDataQueryService(
      */
     fun deleteEntities(entitySetId: UUID, entityKeyIds: Set<UUID>, partitions: Set<Int>): WriteEvent {
         val entitiesByPartition = getIdsByPartition(entityKeyIds, partitions.toList())
-        val hds = dataSourceResolver.resolve(entitySetId)
+        val hds = dataSourceResolver.getDefaultDataSource()
```

Comment:
Why does this need to change? Is it because it's only working with the `ids` table?

Reply:
Correct.